atlp-rwanda/atlp-devpulse-fn
# Ft Documentations Frontend

### PR Description

This PR allows admin to create, update, view and delete docs. 

### Description of tasks that were expected to be completed

As an applicant I can view the documents on the docs page. 
As an admin I should be able to create, view, update and delete the documents on docs page

### How can this been tested?
- Clone repo
- Checkout branch `ft-documentation-fn`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Use apolo server and interact with doc resolvers


### Track PR (issue number & link)
Issue: [#250](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/250)

### Screenshots (If appropriate especially on the frontend)
N/A
